### ISSUE TITLE
Spring4Shell security problem

### ISSUE THREAD

**Issue by gianpaolo-tndigit** [scc-digitalhub/AAC#76]
Spring released emergency updates to fix the 'Spring4Shell' zero-day remote code execution vulnerability, which was leaked prematurely online before a patch was released.

https://spring.io/blog/2022/03/31/spring-framework-rce-early-announcement

VMware products and affected versions:
Spring Framework
- 5.3.0 to 5.3.17
- 5.2.0 to 5.2.19
Older and unsupported versions are also affected

Mitigation
Users of affected versions should apply the following mitigation:
5.3.x users should upgrade to 5.3.18+, 5.2.x users should upgrade to 5.2.20+. No further steps are required.

References:
- https://tanzu.vmware.com/security/cve-2022-22965
- https://www.bleepingcomputer.com/news/security/spring-patches-leaked-spring4shell-zero-day-rce-vulnerability/
- https://www.first.org/cvss/calculator/3.0#CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A : H

**Comment by matteo-s** [comment]
Thanks for the heads up, but the vulnerability can't be exploited on AAC,
mostly because we don't deploy as war on tomcat, but only as spring jar.

See
https://spring.io/blog/2022/03/31/spring-framework-rce-early-announcement#am-i-impacted


Il ven 1 apr 2022, 11:25 gianpaolo-tndigit ***@***.***> ha
scritto:

**Comment by matteo-s** [comment]
Thanks for the heads up, but the vulnerability can't be exploited on AAC,
mostly because we don't deploy as war on tomcat, but only as spring jar.

See
https://spring.io/blog/2022/03/31/spring-framework-rce-early-announcement#am-i-impacted

**Comment by gianpaolo-tndigit** [comment]
I agree with you, only the doubt remains on the phrase "However, the nature of the vulnerability is more general, and there may be other ways to exploit it that have not been reported yet."
